package.json

```diff
@@ -21,10 +21,14 @@
   "bugs": {
     "url": "https://github.com/metarhia/metatests/issues"
   },
-  "main": "./metatests.js",
+  "main": "metatests.js",
   "bin": {
     "metatests": "./bin/cli.js"
   },
+  "files": [
+    "bin/*.js",
+    "lib/*.js"
```

Comment:
Is it possible that there will be files with extensions other than `*.js` in `lib/`?

Reply:
There are no such files now, but we might as well add a whole folder.